{{cookiecutter.github_repository_name}}/{{cookiecutter.app_name}}/users/models.py

```diff
@@ -1,3 +1,4 @@
+{%- if cookiecutter.email_user == "y" %}
```

Comment:
If email_user is not chosen, we should still extend the user model. 
Since it is recommended in the Django docs that: 
[If you’re starting a new project, it’s highly recommended to set up a custom user model, even if the default User model is sufficient for you. This model behaves identically to the default user model, but you’ll be able to customize it in the future if the need arises](https://docs.djangoproject.com/en/2.0/topics/auth/customizing/#using-a-custom-user-model-when-starting-a-project)